### PR TITLE
Fix a number of bugs

### PR DESCRIPTION
**Pull request by calebschoepp**
- Give README a markdown extension
- Handle keys with spaces, slashes or other such characters that are often invalid in HTML IDs or URLs
- Fix white text on white background

### CODE REVIEW

File: explorer/src/index.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ router.get("/internal/kv-explorer/api/stores/:store", async (req): Promise<HttpR
 
 router.get("/internal/kv-explorer/api/stores/:store/keys/:key", async (req): Promise<HttpResponse> => {
     let store = req.params.store;
-    let key = req.params.key;
+    let key = decodeURIComponent(req.params.key);
```
Comment (lann) [inline explorer/src/index.ts:44]: What you want it to "work" "consistently"?

Comment (calebschoepp) [reply explorer/src/index.ts:44]: Can you rephrase, I don't quite understand what you're saying here.

Comment (calebschoepp) [reply explorer/src/index.ts:44]: Oh, I just re-read this and I detected a whiff of sarcasm. Please confirm 😄

**Review by lann — APPROVED** [review]

File: explorer/src/index.ts (hunk 1 above)
Comment (calebschoepp) [reply explorer/src/index.ts:44]: My reading comprehension grows stronger by the day. Within the month I may even begin to understand irony.

**Review by radu-matei — APPROVED** [review]
